pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,65 +1,125 @@
-# How to set up a full node
+# Set Up a Full Node
+Installing and running a full node allows your system to participate in a dYdX chain network.
 
-## Pre-requisite
-1. Linux (Ubuntu Server 22.04.3 recommended)
-2. 8-cpu (ARM or x86_64), 64 GB RAM, 500 GB SSD NVME Storage
+## Prerequisites
+To run a full node, the system that hosts the node must meet the following minimum requirements:
+- Linux (Ubuntu Server 22.04.3 or later recommended)
```

Comment:
nit: I think we have a [hardware req page here](https://docs.dydx.exchange/infrastructure_providers-validators/hardware_requirements). don't think we need to address it as part of this PR but it would be good to think about how to organize such similar info and distinguish: "okay, this is req for running a validator vs this is req for running a non-validating full node"

one option would be to 
1. create two entries under `Hardware Requirements` (validator node vs non-validating full-node)
2. link to the correct section from here

Reply:
agreed, I'd like to address this as part of an info architecture refactor